app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,35 @@
+class InvitationsController < ApplicationController
+  before_action :authenticate!
+  before_action :set_user!
+  before_action :set_invitation!
+
+  def show
+    redirect_to_organization! if @user.student_of? @organization
```

Comment:
Yes, but the link will grow and have redundant info that it's already in the invitation. I was thinking in the student copying the link from a classroom board :stuck_out_tongue: